### PR TITLE
#164198444 Setup react project

### PR DESCRIPTION
**Pull request by adaezeodurukwe**
#### What does this PR do?
Set up react project

#### Description of Task to be completed?
- Download all project dependencies
- Setup webpack
- Setup babel
- Create entry files `App.jsx` and `index.html`

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm run start:dev`

#### What are the relevant pivotal tracker stories?
[#164198444](https://pivotaltracker.com/story/show/164198444)

### CODE REVIEW

**Review by Chrismarcel — COMMENTED** [review]
LGTM

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Authors Haven</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="main.css">
+  <script src="main.js"></script>
```
Comment (andela-gike) [inline public/index.html]: @adaezeodurukwe is there a reason you have a script tag pointing to main.js

**Review by andela-gike — CHANGES_REQUESTED** [review]
Also, don't you think you should have 2 webpack config files; one for dev and one for production

File: public/index.html (hunk 1 above)
Comment (adaezeodurukwe) [reply public/index.html]: I created a standard HTML5 boilerplate to be edited subsequently to suit the project need but I see how it is unnecessary now, I'll fix this. Thanks.

**Review by andela-gike — APPROVED** [review]

**Pull request merged**